frontend/app/rollup.config.js

```diff
@@ -245,9 +244,17 @@ export default {
                     `var parcelRequire;`,
                 ];
                 const cspHashValues = inlineScripts.map(generateCspHashValue);
-                let csp = `script-src 'self' 'unsafe-eval' https://api.rollbar.com/api/ https://platform.twitter.com/ https://www.googletagmanager.com/ ${cspHashValues.join(
-                    " "
-                )}`;
+                let csp = `
+                    style-src * 'unsafe-inline'; 
+                    style-src-elem * 'unsafe-inline';
```

Comment:
Is this needed? Won't it just fall back to `style-src`?

Reply:
Don't really know - it's just copied from the default csp that's recommended so I'm inclined to include it.